introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Tad Peterson.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I enjoyed the Ruby and Git courses.  Next will be the HTML/CSS path.